ecosystem/typescript/sdk_v2/src/bcs/types.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
+export type Seq<T> = T[];
```

Comment:
So I removed this so we don't have any aliased types. Same as `Bytes` <=> `Uint8Array`

Not sure if we want to go either way with it. Personally I like the way `Seq<T>` looks, so I see why jijun originally did it, but it might just confuse people 🤷

Reply:
I had the same thoughts. The Seq<> does looks a bit cleaner, but not sure if this is necessary. We can discuss this offline